inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -263,52 +264,40 @@ void MKLDNNConvolutionNode::getSupportedDescriptors() {
             eltwisePrecision = Precision::FP32;
         }
 
-        if (ndims == 4) {
+        if (one_of(ndims, 4, 5)) {
+            memory::format_tag plainTag = ndims == 4 ? memory::format_tag::nchw : memory::format_tag::ncdhw;
+            memory::format_tag nxcTag = ndims == 4 ? memory::format_tag::nhwc : memory::format_tag::ndhwc;
+            memory::format_tag blockedTag16C = ndims == 4 ? memory::format_tag::nChw16c : memory::format_tag::nCdhw16c;
+            memory::format_tag blockedTag8C = ndims == 4 ? memory::format_tag::nChw8c : memory::format_tag::nCdhw8c;
```

Comment:
Just wondering: why we introduced common names: {ncsp, nspc, nCspxc} but don't use them everywhere and trying to invent smt new?

Reply:
Yes, indeed. Fixed. Thanks.